contracts/atomic-order-example/src/contract.rs

```diff
@@ -32,13 +33,15 @@ pub fn instantiate(
     msg: InstantiateMsg,
 ) -> Result<Response<InjectiveMsgWrapper>, ContractError> {
     let querier = InjectiveQuerier::new(&deps.querier);
-    if let Some(market) = querier.query_spot_market(&msg.market_id)?.market {
+    if let Some(market) = querier.query_spot_market(msg.market_id.clone())?.market {
```

Comment:
I'm wondering whether this is really the best way to go. And by that I mean two things:
* taking ownership (why should the query consume the param?)
* accepting `Into<MarketId>` instead of `MarketId` (or even better `&MarketId`?)

Reply:
Maybe it would be better to create `MarketId` instance as high in the call hierarchy as possible and then just borrow it further downstream to avoid copying it all the time?

Reply:
> accepting Into<MarketId> instead of MarketId

We intentionally did not implement `Into<MarketId>`, see [`c0d8341` (#81)](https://github.com/InjectiveLabs/cw-injective/pull/81/commits/c0d8341fc3f3645e9081ec37a8bfa0d6051b1a08) for reference.